docs/release-notes/NuGet-6.4.md

```diff
@@ -0,0 +1,117 @@
+
+* nuget.exe help command has unlocalized strings - [#12067](https://github.com/NuGet/Home/issues/12067)
+
+* Remove unused localized resources in nuget.exe - [#12066](https://github.com/NuGet/Home/issues/12066)
+
+* [Bug]: NugetSDKResolver doesn't give detailed error messages when it fails - [#12049](https://github.com/NuGet/Home/issues/12049)
+
+* [Bug]: Package signature validation fails on Linux due to missing 'thawte_Primary_Root_CA' in codesignctl.pem - [#12027](https://github.com/NuGet/Home/issues/12027)
+
+* [Bug]: "An item with the same key has already been added" when migrating to CPM with `ProjectDependencies` in solution file - [#12021](https://github.com/NuGet/Home/issues/12021)
+
+* [Bug]: Build failures in dev branch due to renaming of parameter from cpvmEnabled to centralPackageTransitivePinningEnabled  - [#12020](https://github.com/NuGet/Home/issues/12020)
+
+* [Bug]: [Bug Bash] Other versions will lose after selecting a version in the custom version drop-down box for a while - [#11992](https://github.com/NuGet/Home/issues/11992)
+
+* Remove extra layers of abstractions from IVsProjectAdapter, move RuntimeGraph specific methods from VSProject to LegacyPackageReferenceProject - [#11980](https://github.com/NuGet/Home/issues/11980)
+
+* Reduce redundant SolutionDirectory calculation, special-case template wizard solution directory retrieval - [#11936](https://github.com/NuGet/Home/issues/11936)
+
+* Make VS adapter ProjectDirectory sync, use IVsHierarchy only to generate the guids, avoid double casting VSProject4 - [#11928](https://github.com/NuGet/Home/issues/11928)
+
+* [Bug]: NuGet.VisualStudio.Implementation.Extensibility.VsPathContextProvider.TryCreateContext fault - [#11918](https://github.com/NuGet/Home/issues/11918)
+
+* [Bug]: Package version downgrade is not detected due to invalid transitive pinning - [#11760](https://github.com/NuGet/Home/issues/11760)
+
+* _CleanPackageFiles target fails sporadically when (re)building - [#11710](https://github.com/NuGet/Home/issues/11710)
+
+* Avoid calling CreateLockFileTargetLibrary twice when AssetTargetFallback is used - [#11654](https://github.com/NuGet/Home/issues/11654)
+
+* Package source mapping should check for duplicate node keys - [#11573](https://github.com/NuGet/Home/issues/11573)
+
+* VSSolutionManager.DoesNuGetSupportsAnyProjectAsync can exit at the first supported projec - [#11555](https://github.com/NuGet/Home/issues/11555)
+
+* Review all sync ServiceLocator calls and move to async where possible - [#11203](https://github.com/NuGet/Home/issues/11203)
+
+* [Bug Bash]The new designs of hovered-on menu between VS and NuGet are inconsistent - [#10978](https://github.com/NuGet/Home/issues/10978)
+
+* [Bug]: Metadata like PrivateAssets does not flow from parent to transitively pinned dependency in CPM - [#10311](https://github.com/NuGet/Home/issues/10311)
+
+**[List of commits in this release](https://github.com/NuGet/NuGet.Client/compare/6.3.1.1...6.4.0.117)**
```

Comment:
```suggestion
**[List of commits in this release](https://github.com/NuGet/NuGet.Client/compare/6.3.1.1...6.4.0.123)**
```

The reason for this suggestion is mentioned https://github.com/NuGet/Client.Engineering/issues/1886#issuecomment-1306008269

Reply:
how about `6.3.1.1..release-6.4.x`, so we don't need to update it every time we service the branch?

Reply:
The tool that I have just uses the latest tags. 117 was the latest tag when I ran it.

I prefer the exact versions, but whatever we decide, we should just make sure we update  the automation.